actions/promote_package.yaml

```diff
@@ -0,0 +1,40 @@
+---
+name: promote_package
+runner_type: mistral-v2
+description: Promote a package to production from staging.
+enabled: true
+entry_point: workflows/promote_package.yaml
+parameters:
+  package:
+    type: string
+    description: Package to promote
+    required: true
+  version:
+    type: string
+    description: Version to promote
+    required: true
+  revision:
+    type: string
+    description: Revision to promote
+    required: true
+  os:
+    description: OS to promote (will promote all if not specified)
+    required: true
+    default: all
```

Comment:
That's ok. That's why there's a choice of OS (or rpm/deb) available, and also why the workflow won't fail hard on upload errors: it'll show that one promotion succeeded and the others failed, nothing will break. It would be nice to make it smarter in the future, of course.